src/styles/_mixins.scss

```diff
@@ -36,7 +38,7 @@
 @mixin textarea-default {
   background-color: var(--KInputBackground, var(--white, #ffffff));
   box-shadow: none !important;
-  outline: 1px solid var(--KInputBorder, var(--grey-300, #e7e7ec)) !important;
+  // outline: 1px solid var(--KInputBorder, var(--grey-300, #e7e7ec)) !important;
```

Comment:
question: why is this line commented out?

Reply:
Ended up removing this because it's now overwritten by another mixin.